src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -890,6 +890,16 @@ public AppPage verifyHtmlMainContent(String filePath) throws IOException {
     }
     
     /**
+     * Verifies that the title of the loaded page is the same as {@code expectedTitle}
+     * @param expectedTitle
+     * @return The page (for chaining method calls).
+     */
+    public AppPage verifyTitle(String expectedTitle) {
+        assertEquals(expectedTitle, browser.driver.getTitle());
+        return this;
```

Comment:
You don't really need to `return this;`. It doesn't really make sense for any verification method to have a return value.

P.S I know the others have, and I'm quite itching to fix them :p